### PR TITLE
Create fp-020-responsiveness.md

### LINKED ISSUE #960: PROPOSED Principle #20 "Responsiveness"
Initial thoughts/text:

Name: Responsiveness to community term requests

(text moved from #10) Recommendation:It is required that ontology developers offer channels for community participation. A discussion mailing list and issue tracker are required to obtain an OBO Foundry PURL. It is recommended to aim for a good level of responsiveness to community requests and suggestions, which is also related to the maintenance principle (#16). 

Issue tracker:        https://github.com/geneontology/go-ontology/issues/ 
Discussion list:     go-discuss@mailman.stanford.edu
Collaboration of this sort can be demonstrated by having an active discussion mailing list, responsiveness to community requests

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
This is just a draft, needs review.

I wasn't sure what to put under <Category:> I added InProgress but not sure if that is a valid annotation.

closes #960

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@nataled should this principle have info about having a mandatory tracker as well?

**Comment by nataled** [comment]
@nicolevasilevsky yes. Please look at the draft notes given in #960 for the Responsiveness principle (the number for this principle is 20, not 17)

I'm not sure if what you're doing with the .md file will render it public right away, but it has several steps to go through before that should happen.

**Comment by nicolevasilevsky** [comment]
@nataled thanks for the feedback, I made those changes.

I think this page would be live once this PR is merged, would you like me to move this doc somewhere else?

**Comment by nataled** [comment]
@nicolevasilevsky yes, please. We've never done the wordsmithing while live. In the past we used a Google doc, but then (as you can see) we moved toward using a github tracker item (which, honestly, is a much more difficult method). Only after the wording was in place was the md file made.

**Comment by nicolevasilevsky** [comment]
- [ ] need to coordinate with technical working group to create automatic check

**Comment by nicolevasilevsky** [comment]
see draft in google doc here:
https://docs.google.com/document/d/1nCCO88uOWgWp3z5AwyapcTP0T_C2ZmdPi7cWiO9_aDM/edit#

**Comment by jamesaoverton** [comment]
The changes required to the automated checks and Dashboard are simple enough. I would wait until the Principle is finalized, and I would ask @matentzn to update the checks and dashboard at that point.

**Comment by nataled** [comment]
Thanks @jamesaoverton we'll keep you informed.

**Comment by cmungall** [comment]
change the name of the file. Prefix with `fp-020-`

please keep the proposed changes in the PR so it is transparent to all

you can merge the PR and it will not show up unless explicitly linked

File: principles/fp-020-responsiveness.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+-------
+Ontology development benefits from community input, which is strongly encouraged by the OBO Foundry. Accordingly, "responsiveness" is a key quality of our general collaborative spirit. This principle is intended to guide the processes to ensure community responsiveness. 
+
+Recommendation
+------- 
+Ontology developers SHOULD aim for a good level of responsiveness to community requests and suggestions.
+
+Expectations of responsiveness:
+1. Issues are contributions - and should be met by a thankful attitude. When receiving a ticket on your issue tracker, the first thing to do is thank the contributor, even if a ticket cannot be addressed right away. 
+1. If aa ticket cannot be addressed right away, eplain when you plan to look at the ticket in more detail. This should happen soon after a ticket is issued - ideally in the first 2-5 working days after the issue entered your tracker.
```
Comment (nlharris) [inline principles/fp-020-responsiveness.md:27]: typo here, "aa ticket".

File: principles/fp-020-responsiveness.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Recommendation
+------- 
+Ontology developers SHOULD aim for a good level of responsiveness to community requests and suggestions.
+
+Expectations of responsiveness:
+1. Issues are contributions - and should be met by a thankful attitude. When receiving a ticket on your issue tracker, the first thing to do is thank the contributor, even if a ticket cannot be addressed right away. 
+1. If aa ticket cannot be addressed right away, eplain when you plan to look at the ticket in more detail. This should happen soon after a ticket is issued - ideally in the first 2-5 working days after the issue entered your tracker. 
+1. Please do not close tickets without responding. If a ticket is out of scope for a repository, recommend moving it to a different repository.
+1. It is recommended that someone from each ontology is responsible for triaging issues and requests.
```
Comment (nlharris) [inline principles/fp-020-responsiveness.md:29]: is responsible -> be responsible

File: principles/fp-020-responsiveness.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+Implementation
+-------
+A discussion mailing list and issue tracker are required to obtain an OBO Foundry PURL. The OBO Foundry offers the following recommendations on the responsiveness in issue trackers (GitHub is recommended).
+
+1. Establish an issue tracker (GitHub is recommended).
+2. Establish a mailing list (For example, google groups).
+3. The issue tracker and mailing list MUST be monitored and responded to in a reasonable amount of time (ideally 2-5 working days).
```
Comment (nlharris) [inline principles/fp-020-responsiveness.md:37]: so MUST be responded to in a reasonable amount of time, but that amount of time is only suggested, not required?

**Review by nlharris — APPROVED** [review]

**Comment by nlharris** [comment]
Where are we with this? We just need a few more reviewers to approve it?

**Comment by nataled** [comment]
Seems so. Based on an explanation provided by Chris that this will not be public until a link is established, I have approved the file even though more revisions are required (in progress within the EWG). I ask @matentzn to also give his approval.

**Review by nataled — APPROVED** [review]

**Comment by nlharris** [comment]
Thanks. Are there points that should be discussed in tomorrow's OBO Operations meeting?

**Comment by nataled** [comment]
We are not yet at that point of the process.

**Pull request merged**